ui/src/components/microsegmentation/AddStaticInstances.js

```diff
@@ -0,0 +1,278 @@
+/*
+ * Copyright 2020 Verizon Media
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import Input from '../denali/Input';
+import InputLabel from '../denali/InputLabel';
+import Member from '../member/Member';
+import styled from '@emotion/styled';
+import { colors } from '../denali/styles';
+import AddModal from '../modal/AddModal';
+import RequestUtils from '../utils/RequestUtils';
+import InputDropdown from '../denali/InputDropdown';
+import Icon from '../denali/icons/Icon';
+
+const Resourcetypes = [
+    {
+        name: 'IP Address',
+        value: 'IP Address',
+    },
+];
+
+const SectionDiv = styled.div`
+    align-items: center;
+    display: flex;
+    flex-flow: row nowrap;
+    padding: 10px 10px;
+`;
+
+const StyledInputLabel = styled(InputLabel)`
+    float: left;
+    font-size: 14px;
+    font-weight: 700;
+    padding-top: 12px;
+    width: 20%;
+`;
+
+const StyledInput = styled(Input)`
+    width: 350px;
+    padding: 0 12px;
+`;
+
+const AddCircleDiv = styled.div`
+    margin-top: 5px;
+    margin-left: 5px;
+`;
+
+const StyledIncludedMembersDiv = styled.div`
+    width: 65%;
+`;
+
+const SectionsDiv = styled.div`
+    width: 780px;
+    text-align: left;
+    background-color: ${colors.white};
+`;
+
+export default class AddStaticInstances extends React.Component {
+    constructor(props) {
+        super(props);
+        this.api = props.api;
+        this.addMember = this.addMember.bind(this);
+        this.onSubmit = this.onSubmit.bind(this);
+        this.resourceTypeChanged = this.resourceTypeChanged.bind(this);
+        this.inputChanged = this.inputChanged.bind(this);
+        this.state = {
+            members: [],
+            resourceValue: '',
+        };
+    }
+
+    resourceTypeChanged(chosen) {
+        if (chosen && chosen.value != null) {
+            this.setState({
+                resourceType: chosen.value,
+            });
+        }
+    }
+
+    addMember() {
+        let resourceVal = this.state.resourceValue;
+        let members = this.state.members;
+
+        if (!resourceVal) {
+            return;
+        }
+        let names = (resourceVal || '')
```

Comment:
can we move this code to NameUtils.js and replace it everywhere this code is being used?